pkg/csi/service/common/prometheus/metrics.go

```diff
@@ -38,6 +40,31 @@ const (
 	// PrometheusDetachVolumeOpType represents the DetachVolume operation.
 	PrometheusDetachVolumeOpType = "detach-volume"
 
+	// CNS operation types
+
+	// PrometheusCnsCreateVolumeOpType represents the CreateVolume operation.
+	PrometheusCnsCreateVolumeOpType = "create-volume"
+	// PrometheusCnsDeleteVolumeOpType represents the DeleteVolume operation.
+	PrometheusCnsDeleteVolumeOpType = "delete-volume"
+	// PrometheusCnsAttachVolumeOpType represents the AttachVolume operation.
+	PrometheusCnsAttachVolumeOpType = "attach-volume"
+	// PrometheusCnsDetachVolumeOpType represents the DetachVolume operation.
+	PrometheusCnsDetachVolumeOpType = "detach-volume"
+	// PrometheusCnsUpdateVolumeMetadataOpType represents the UpdateVolumeMetadata operation.
+	PrometheusCnsUpdateVolumeMetadataOpType = "update-volume-metadata"
+	// PrometheusCnsExpandVolumeOpType represents the ExpandVolume operation.
+	PrometheusCnsExpandVolumeOpType = "expand-volume"
```

Comment:
Why don't we have `PrometheusExpandVolumeOpType` for CSI operations ? Is that coming in upcoming PRs ?

Reply:
Yes, it will be in a separate PR. Note that there are many more PRs to come to expose metrics.